pages/ksphere/konvoy/latest/release-notes/kubernetes-base-addon/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: layout.pug
+navigationTitle: Kubernetes Base Addons
+title: Kubernetes Base Addons
+menuWeight: 0
+beta: true
+excerpt: View release-specific information for Kubernetes base addons
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+## Kubernetes Base Addons
+
+### March 27, 2020
+
+[stable-1.16-1.4.1](https://github.com/mesosphere/kubernetes-base-addons/releases/tag/stable-1.16-1.4.1), [stable-1.15-1.4.1](https://github.com/mesosphere/kubernetes-base-addons/releases/tag/stable-1.15-1.4.1)
+
+-   Velero:
+    - Revert the velero refactor to stable-1.16-1.4.0 due to an instability issue.
+
+-   Velero-minio:
+    - Fix instability issues after completing upgrade.
+
+### March 25, 2020
+
+[stable-1.16-1.4.0], [stable-1.15-1.4.0]
```

Comment:
I know we agreed on removing the links this morning but this is different from what exists in konvoy docs. Please make sure to sync them otherwise next time we run the sync script this will get overwritten